spec/warren/config/consumers_spec.rb

```diff
@@ -31,23 +31,48 @@
   end
 
   describe '#add_consumer' do
-    subject(:add_consumer) do
-      consumers.add_consumer('name', desc: 'description', queue: 'queue_name',
-                                     bindings: Configuration.topic_exchange_bindings,
-                                     subscribed_class: 'Warren::Subscriber::Name')
-    end
+    context 'with no delay exchange' do
+      subject(:add_consumer) do
+        consumers.add_consumer('name', desc: 'description', queue: 'queue_name',
+                                       bindings: Configuration.topic_exchange_bindings,
+                                       subscribed_class: 'Warren::Subscriber::Name',
+                                       delay: 0)
```

Comment:
Assume `delay` is required/ could there be a default delay of 0?

Reply:
If you want to requeue something immediately you have the requeue option. I can't see a reason why you'd want a delay of zero. If you did though, you'd still be able to specify it manually in the `warren_consumers.yml`.